### PR TITLE
Add contact information for safety officers

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -28,7 +28,11 @@ Please note that the list above is alphabetised and is therefore not ranked in a
 
 An up-to-date list of members of the nf-core core team can be found [here](https://nf-co.re/about).
 
-Our Safety Officers are Saba Nafees, Cris Tuñí, and Michael Heuer.
+Our Safety Officers are 
+
+- [@ctuni](https://github.com/ctuni): Cris Tuñí ([contact](mailto:c.tuni.dominguez@gmail.com))
+- [@heuerm](https://github.com/heuermh): Michael Heuer ([contact](mailto:mheuer@mammothbiosci.com))
```
Comment (heuermh) [inline CODE_OF_CONDUCT.md:34]: [@heuerm] &rarr; [@heuermh]

**Review by heuermh — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (JoseEspinosa) [reply CODE_OF_CONDUCT.md:34]: Ups 🙏

File: markdown/about.md [hunk 2]
```diff
@@ -86,9 +86,9 @@ The safety team is responsible for ensuring the community is a safe place and re
 
 The current safety team in 2023 are:
 
-- [@ctuni](https://t.co/mQInuSXc5R) Cris Tuñí
-- [@heuerm](https://github.com/heuermh) Michael Heuer
-- [@snafees](https://github.com/snafees/) Saba Nafees
+- [@ctuni](https://github.com/ctuni): Cris Tuñí ([contact](mailto:c.tuni.dominguez@gmail.com))
+- [@heuerm](https://github.com/heuermh): Michael Heuer ([contact](mailto:mheuer@mammothbiosci.com))
```
Comment (jfy133) [inline markdown/about.md:90]:
```suggestion
- [@heuermh]](https://github.com/heuermh): Michael Heuer ([contact](mailto:mheuer@mammothbiosci.com))
```

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Pull request merged**